mitodl/mitxonline
# Featured Products Carousel

# What are the relevant tickets?
Updates #https://github.com/mitodl/hq/issues/1677

# Description (What does it do?)
Adds programs to featured products in Wagtail
Adds feature flagging to be able to turn these changes off and on depending on environment
New Styling & carousel for Featured Products


# Screenshots (if appropriate):
![Desktop](https://github.com/mitodl/mitxonline/assets/7756053/58d4f5d2-b7f9-46da-9859-47b7f0a51699)

![Mobile](https://github.com/mitodl/mitxonline/assets/7756053/a9988945-144a-4dc4-851e-f7eee5e9f9b9)

# How can this be tested?
Make sure that you have a number of courses and programs set up
Go into the wagtail admin and add some of each to your Featured products on the home page
If in dev, this should automatically be on.
Look at both desktop and mobile and manipulate both.

# Additional Context
The flag is called mitxonline-new-featured-carousel in post-hog. Let me know if you need access.
[Figma Link](https://www.figma.com/file/Gs4zIhOFv5gVvafawwl6PF/MITx-Online?type=design&node-id=558%3A2708&mode=design&t=REGKyMeHdusZuvU1-1)
